yuca/utils.py

```diff
@@ -179,6 +179,7 @@ def make_gif(frames_path="./assets/gif_frames/", gif_path="./assets", \
 def save_fig_sequence(grid, ca, num_steps=10, \
         frames_path="./assets/gif_frames", \
         gif_path="./assets", mode=0, tag="no_tag", \
+        speedup=1,\
```

Comment:
Particularly useful in reaction diffusion systems (which have slow dynamics), speedup simulates multiple time steps per frame.